tests/test_list

```diff
@@ -3,7 +3,9 @@ test_fiod.sh
 test_fs-drift.sh
 test_smallfile.sh
 test_pgbench.sh
+test_fiod.sh
```

Comment:
this is added again ? its already at line2

Reply:
fixed